sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerConfig.java

```diff
@@ -433,4 +436,19 @@ public boolean isRegionAwarenessEnabled() {
     public String getSchedulerIP() {
         return envStore.getRequired(LIBPROCESS_IP_ENVVAR);
     }
+
+    public JSONObject getBuildInfo() {
+        JSONObject jsonObject = new JSONObject();
+        jsonObject.put(PACKAGE_NAME_ENV, getPackageName());
+        jsonObject.put(PACKAGE_VERSION_ENV, getPackageVersion());
+        jsonObject.put(PACKAGE_BUILD_TIME_EPOCH_MS_ENV, Instant.ofEpochMilli(getPackageBuildTimeMs()));
+        for (Field f : SDKBuildInfo.class.getDeclaredFields()) {
```

Comment:
Rather than using reflection, I think it's fine to just statically define some names here. I could see someone being suprised by these names changing if SDKBuildInfo members are renamed.